tests/factories.py

```diff
@@ -21,11 +21,14 @@
 
 
 class RecommendationFactory(factory.Factory):
-    """ Creates fake pets that you don't have to feed """
+    """ Creates fake recommendations that you don't have to feed """
 
     class Meta:
         model = Recommendation
 
     product_id1 = FuzzyChoice(choices=[1, 2, 3, 4])
     product_id2 = FuzzyChoice(choices=[5, 6, 7, 8])
     relationship = FuzzyChoice(choices=[Type.UP_SELL, Type.CROSS_SELL, Type.ACCESSORY])
+    title = FuzzyChoice(choices=["Recommendation one", "Recommendation two", "Recommendation three", "Recommendation four"])
+    recommendation_type = FuzzyChoice(choices=["Upsell", "Cross_sell", "Accessory", "Bundle"])
+    active = FuzzyChoice(choices=[True, False])
```

Comment:
I don't see these fields added to the model and I am getting compiling error when running nosetests in list branch. Have you pushed all of your commits? Also recommendation_type seems to be the same thing as relationship.

Reply:
yea im kinda confused because when i ran nosetests i got compiling errors.